modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/services/rest/TestrayTask.ts

```diff
@@ -106,20 +106,30 @@ class TestrayTaskImpl extends Rest<TaskForm, TestrayTask, NestedObjectOptions> {
 	}
 
 	public async assignTo(task: TestrayTask, userIds: number[]) {
-		await this.update(task.id, {
+		const response = await this.update(task.id, {
 			dueStatus: TaskStatuses.IN_ANALYSIS,
 			name: task.name as string,
 		});
+
 		await this.assignUsers(task.id, userIds);
+
+		return response;
 	}
 
 	public async abandon(task: TestrayTask) {
-		return this.update(task.id, {
+		return await this.update(task.id, {
```

Comment:
Can you remove the async in the function?

It's not necessary, the function already returns a promise, no need for `await and async` in this case

```suggestion
		return this.update(task.id, {
```